### PR TITLE
#1496 basic prescription page

### LINKED ISSUE #1496: Basic prescriptions page
## Is your feature request related to a problem? Please describe.

Can't seem to get a start on dispensing

## Describe the solution you'd like

A basic prescriptions+prescription page, something to work with

## Describe alternatives you've considered

Not doing this

## Additional context

This is a simple, basic page to get started and start iterating on. By having a basic UI, other core/backend logic can be worked on which needs less consensus/discussion than UI

### PR DESCRIPTION
**Pull request by joshxg**
Fixes #1496 

## Change summary

- Very basic prescription + prescriptions page

image

image

### Related areas to think about

- This does sync to and from desktop, finalises and saves the correct fields

### CODE REVIEW

File: src/navigation/actions.js [hunk 1]
```diff
@@ -30,6 +30,44 @@ import { ROUTES } from './constants';
  *
  */
 
+/**
+ * Action creator which first creates a customer invoice, and then navigates to it
```
Comment (mark-prins) [inline src/navigation/actions.js]: should that be prescription?

File: src/widgets/modals/DataTablePageModal.js [hunk 2]
```diff
@@ -71,7 +71,30 @@ const DataTablePageModalComponent = ({
       case MODAL_KEYS.TRANSACTION_COMMENT_EDIT:
       case MODAL_KEYS.REQUISITION_COMMENT_EDIT:
         return <TextEditor text={currentValue} onEndEditing={onSelect} />;
-
+      case MODAL_KEYS.SELECT_PRESCRIBER:
+        return (
+          <AutocompleteSelector
+            options={UIDatabase.objects('Prescriber')}
+            isOpen={isOpen}
+            placeholderText="Select a prescriber"
```
Comment (mark-prins) [inline src/widgets/modals/DataTablePageModal.js]: do translations for these come later?

File: src/navigation/actions.js (hunk 1 above)
Comment (joshxg) [reply src/navigation/actions.js]: 1ee9f53

File: src/widgets/modals/DataTablePageModal.js (hunk 2 above)
Comment (joshxg) [reply src/widgets/modals/DataTablePageModal.js]: yeah I guess 🤣

**Pull request merged**